documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1051 @@
+endif::[]
++
+
+. Create a `StorageMap` manifest to map source and destination storage:
++
+ifdef::vmware[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storage_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode> <1>
+      source:
+        id: <source_datastore> <2>
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode>
+      source:
+        id: <source_datastore>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
+<2> Specify the VMware data storage MOR. For example, `f2737930-b567-451a-9ceb-2887f6207009`.
+endif::[]
+
+ifdef::rhv[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storage_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode> <1>
+      source:
+        id: <source_datastore> <2>
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode>
+      source:
+        id: <source_datastore>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
+<2> Specify the {rhv-short} storage domain UUID. For example, `f2737930-b567-451a-9ceb-2887f6207009`.
+endif::[]
+
+ifdef::ova[]
+[NOTE]
+====
+For OVA, the `StorageMap` can map only a single storage, which all the disks from the OVA are associated with, to a storage class at the destination. For this reason, the storage is referred to in the UI as "Dummy storage for source provider <provider_name>".
+====
++
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storage_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode> <1>
+      source:
+        name: <source_datastore> <2>
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode>
+      source:
+        name: <source_datastore>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
+<2> Specify the TBD.
+endif::[]
```

Comment:
@bkhizgiy What is TBD here?

Reply:
@RichardHoch I'm not sure actually, seams to be sort of a place holder:/

Reply:
@ahadas What do you think?

Reply:
@ahadas 21 February -- suggestions? comment?

Reply:
@ahadas Just to be clear... if I write  name: "Dummy storage for source provider <provider_name>" , then the user will include the quotation marks in the YAML. Is that what we want?

Reply:
@RichardHoch you're right, no need for the quotes but just to make it clear that `<provider_name>` should be replaced with the name of the provider